public/app/features/dashboard/components/ShareModal/SharePublicDashboard/Configuration.tsx

```diff
@@ -11,14 +11,18 @@ import { useIsDesktop } from 'app/features/dashboard/utils/screen';
 import { getTimeRange } from 'app/features/dashboard/utils/timeRange';
 
 export const Configuration = ({
+  annotationsEnabled,
```

Comment:
I would consider the same consistency with isPubDashEnabled, so I would put isAnnotationsEnabled